site/frontend/src/pages/compare/compile/table/benchmark-detail.vue

```diff
@@ -215,6 +243,42 @@ onMounted(() => renderGraph());
       </div>
       <div ref="chartElement"></div>
     </div>
+    <div class="links">
+      <div class="title bold">Links</div>
+      <ul>
+        <li>
+          <a
+            :href="graphLink(props.artifact, props.metric, props.testCase)"
+            target="_blank"
+          >
+            History graph
```

Comment:
Maybe we could name that link "history: 30 days up to commit" or something, so that we can have links to graphs up to the commit, and without an end date i.e. up to now -- to easily see the future behavior after the PR with the possibility to change the graph kind.

(This doesn't need to be in this PR.)

Reply:
Added to my TODO list.

Reply:
I have a local branch doing it locally, and will open a PR soon after this one lands.